pkg/cmd/openshift-apiserver/server.go

```diff
@@ -48,7 +50,10 @@ func RunOpenShiftAPIServer(masterConfig *configapi.MasterConfig) error {
 	if err != nil {
 		return err
 	}
-	if err := informers.AddUserIndexes(); err != nil {
+
+	if err := informers.GetInternalOpenshiftUserInformers().User().V1().Groups().Informer().AddIndexers(cache.Indexers{
```

Comment:
@tnozicka it is just removing extra helper which forbids us to return interface instead of local struct. This index should be added to the informer directly or in place where it is actually being used...